dcp/DcpHandler.go

```diff
@@ -290,17 +290,17 @@ func (dh *DcpHandler) FlushCollection(flush gocbcore.DcpCollectionFlush) {
 
 func (dh *DcpHandler) CreateScope(creation gocbcore.DcpScopeCreation) {
 	// Overloading collectionID field for scopeID because differ doesn't care
-	dh.writeToDataChan(CreateMutation(creation.VbID, nil, creation.SeqNo, 0, 0, 0, 0, gomemcached.DCP_SYSTEM_EVENT, nil, 0, creation.ScopeID))
+	dh.writeToDataChan(CreateMutation(creation.VbID, nil, creation.SeqNo, 0, 0, 0, 0, gomemcached.DCP_SYSTEM_EVENT, nil, 0, creation.ScopeID, nil, nil))
```

Comment:
These function signatures don't need to change per my comment above

Reply:
As discussed, we don't change the function signature of `dh.writeToDataChan` . We are just passing the additional required parameters to the constructor of the Mutation struct.